src/solution.c

```diff
@@ -746,6 +771,11 @@ void solution_setup()
 
   SETTING("sbp", "obs_msg_max_size", msg_obs_max_size, TYPE_INT);
 
+  SETTING("nmea", "gpgsv_msg_rate", gpgsv_msg_rate, TYPE_INT);
+  SETTING("nmea", "gprmc_msg_rate", gprmc_msg_rate, TYPE_INT);
+  SETTING("nmea", "gpvtg_msg_rate", gpvtg_msg_rate, TYPE_INT);
+  SETTING("nmea", "gpgll_msg_rate", gpgll_msg_rate, TYPE_INT);
```

Comment:
Again, could move these to a setup function in nmea.c (if it doesn't have one already)